src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -171,6 +172,14 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
   /// This is technically not draining a node. It should be just called "kill node".
   virtual void DrainNode(const NodeID &node_id);
 
+  /// Update node state from a resource view sync message.
+  ///
+  /// \param node_id The ID of the node to update.
+  /// \param resource_view_sync_message The sync message containing the new state and
+  /// resource data.
```

Comment:
let's document the semantic that if the node is not already in `alive_nodes_`, the update will be ignored. I would not have expected that without looking at the implementation of `UpdateNode`

Reply:
Makes sense. I have updated the comment and the function name to `UpdateAliveNode` to align with `GetAliveNode`.